x/bank/keeper/keeper.go

```diff
@@ -20,6 +20,7 @@ var _ Keeper = (*BaseKeeper)(nil)
 // between accounts.
 type Keeper interface {
 	SendKeeper
+	WithMintCoinsRestriction(MintingRestrictionFn) BaseKeeper
```

Comment:
```suggestion
```

if you remove this, it's not api breaking. People that need that function can cast